README.md

```diff
@@ -32,7 +32,76 @@ For good first issues regarding contributing, see [issues labeled with open sour
 
 # Local development environment
 
-Please contribute by writing a document about local development environment. See above.
+This section is still under construction, if you have troubles setting the environment, please contact us (see below section).
+
+## Prerequisites
+
+Project requires Node.js >=12 <13 to be installed; all other dependencies come from NPM via package.json.
+
+You will need Node and npm (npm comes with Node). We recommend [using a node version manager, nvm](https://github.com/nvm-sh/nvm) to install Node, as it allows automatically switching between node versions across projects.
+
+We recommend also installing [direnv](https://direnv.net/), which helps automatically set environment variables as you switch directories.
+
+## Development environment setup
+
+Clone the project from GitHub (make sure to clone it from the directory you wish to contain your project)
+
+```shell
+git clone git@github.com:futurice/symptomradar.git
+```
+
+Install NPM dependencies
+
+```shell
+cd symptomradar
+rm -rf node_modules
+nvm use
+npm i
+```
+
+If you have some issues with the packages installation, please try the following:
+
+```shell
+rm -rf node_modules
+nvm use
+npm i --no-optional
```

Comment:
Thank you @sjarva for sharing this one neat trick!

Reply:
Haha, you're welcome 😇